### DISCUSSION TITLE
OsmAnd (nightly) doesn't have Indonesian selectable in Languages

### DISCUSSION THREAD

**Discussion by Linerly** [osmandapp/OsmAnd#14150] [General]
I've translated OsmAnd to Indonesian for quite a while (the Android one on Weblate is 74%, Phrases is 28%), but I tried downloading the nightly version of OsmAnd from the website a while ago, and I noticed that Indonesian still isn't there even though it's mostly translated, I just want to see how the translation looks like in OsmAnd

**Comment by vshcherb** [comment]
You still should be able to select Indonesian in your device settings and the application should support Indonesian translation then. In OsmAnd Settings it should be Global Settings -> Language -> Use system language.

Anyway We will add today Indonesian to the list

**Reply by Linerly** [reply]
Ah, okay then

**Reply by Linerly** [reply]
Just downloaded the nightly, it has Indonesian selectable in Languages, but there's no actual translation in the app

**Reply by Linerly** [reply]
![Screenshot 2022-10-25 at 11-35-09 OsmAnd_Android — Indonesia](https://user-images.githubusercontent.com/47475676/197763392-bac55253-3f1a-40ea-b2bf-6368bfe5af92.png)
I've finished the Indonesian translations just now…

![Screenshot 2022-10-25 at 11-36-55 OsmAnd_Phrases — Indonesia](https://user-images.githubusercontent.com/47475676/197763636-263791ed-5273-4655-ac0d-1d2ef9aa0a5c.png)
…but is the Phrases one also necessary?

**Reply by vshcherb** [reply]
Phrases are translations of POI types so you can see they wouldn't have translations

**Reply by Linerly** [reply]
I've recently updated OsmAnd to 4.3.3 and I am now seeing the translations — thank you very much!

The only thing left to do is to finish translating the Phrases
